Fix building/linking problem as reported in: http://bugs.gentoo.org/334891

Not reported to upstream because the patch in upstream bug[1] should be applied instead.

[1] http://bugzilla.xfce.org/show_bug.cgi?id=6575

--- configure.ac
+++ configure.ac
@@ -31,6 +31,7 @@
 
 dnl configure the panel plugin
 XDT_CHECK_PACKAGE([LIBXFCE4PANEL], [libxfce4panel-1.0], [4.3.20])
+XDT_CHECK_PACKAGE([LIBXFCEGUI4], [libxfcegui4-1.0], [4.3.20])
 
 dnl translations
 XDT_I18N([be ca cs en_GB eu fr hu nb_NO pl pt_PT sq uk])
--- panel-plugin/Makefile.am
+++ panel-plugin/Makefile.am
@@ -3,10 +3,12 @@
 
 xfce4_cpugraph_plugin_CFLAGS =					\
 	-DPACKAGE_LOCALE_DIR=\"$(localedir)\"			\
-	@LIBXFCE4PANEL_CFLAGS@
+	@LIBXFCE4PANEL_CFLAGS@					\
+	@LIBXFCEGUI4_CFLAGS@
 
 xfce4_cpugraph_plugin_LDADD =					\
-	@LIBXFCE4PANEL_LIBS@
+	@LIBXFCE4PANEL_LIBS@					\
+	@LIBXFCEGUI4_LIBS@
 
 xfce4_cpugraph_plugin_SOURCES =					\
 	mode.c		\
